R/chat.R

```diff
@@ -535,8 +541,127 @@ print.ChatOpenAI <- function(x, ...) {
   invisible(x)
 }
 
+#' @export
+#' @examplesIf elmer:::openai_key_exists()
+#' chat <- new_chat_openai(echo = TRUE)
+#' chat$chat(
+#'   list(
+#'     "What do you see in these images?",
+#'     content_image_url("https://www.r-project.org/Rlogo.png"),
+#'     content_image_file(system.file("httr2.png", package = "elmer"))
+#'   )
+#' )
```

Comment:
Do we want to use `...`? e.g.

```suggestion
#' chat$chat(
#'     "What do you see in these images?",
#'     content_image_url("https://www.r-project.org/Rlogo.png"),
#'     content_image_file(system.file("httr2.png", package = "elmer"))
#' )
```

Reply:
That's probably better, yeah. Do I use `rlang::list2(...)`?